resources/views/timelines/_timeline.blade.php

```diff
@@ -94,11 +94,32 @@ class="btn btn-box-tool" role="button"
     @endcan
     </ul>
 @empty
+    @php
+    $position = 0;
+    @endphp
     <div class="alert alert-warning">
-        <p>{{ __('timelines.helpers.no_era') }}</p>
+        <p>{{ __('timelines.helpers.no_era') }} </p>
+        @can('update', $timeline)
+        <br>  
```

Comment:
Instead of an empty br (looks ugly when copy pasting), add the "mb-2" class to the p above. This will add a margin-bottom of 0.5rem